src/server/page.ts

```diff
@@ -33,7 +32,7 @@ const clientConfig: ClientConfig = {
   adyenEnvironment: ADYEN_ENVIRONMENT,
   adyenOriginKey: ADYEN_ORIGIN_KEY,
   contentServiceEndpoint: CONTENT_SERVICE_ENDPOINT,
-  giraffeEndpoint: GIRAFFE_ENDPOINT,
+  giraffeEndpoint: '/new-member/graphql',
```

Comment:
Got stuck on this, but then I saw that you are going to update the heruko env. 👍